File: docker_compose/feinplanung/HiCuMes.Persistence/Entities/Mappingrule.cs
using System;
using System.Collections.Generic;

namespace HiCuMes.Persistence.Entities;

public partial class Mappingrule
{
    public long Id { get; set; }

    public string? InputSelector { get; set; }

    public string? OutputSelector { get; set; }

    public string? Rule { get; set; }

    public long UiId { get; set; }
}

File: schemaMapper/mappingEngine/src/main/java/de/iisys/sysint/hicumes/mappingEngine/models/mapping/MappingRule.java
package de.iisys.sysint.hicumes.mappingEngine.models.mapping;

import lombok.AllArgsConstructor;
import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Data
@AllArgsConstructor
@Entity
public class MappingRule {
    @Id
    @GeneratedValue
    protected long id;

    private long uiId;
    @Column(length = 1020)
    private String rule;
    private String inputSelector;
    private String outputSelector;


    public MappingRule() {
    }
}
